src/confluent/azext_confluent/custom.py

```diff
@@ -0,0 +1,17 @@
+# --------------------------------------------------------------------------
```

Comment:
Should custom.py be in generated or manual folder?

Reply:
I did not modify this file after it's autogenerated, so it's only in the generated folder.